caPutLogApp/caPutLogTask.h

```diff
@@ -18,6 +18,8 @@ extern "C" {
 #define MAX_ARRAY_SIZE_BYTES 0
 #endif
 
+#define DEFAULT_BURST_TIMEOUT 5.0
```

Comment:
I find it a bit odd that the json/non-json default timeouts are defined separately. Couldn't they both be but in one place? Note in particular that both `caPutLogTask.c` and `caPutJsonLogTask.cpp` include this file...

Reply:
Agree, better to leave this one than inside the class, so it's same for both